ega-charts/localega/templates/mapper-deploy.yaml

```diff
@@ -25,27 +25,44 @@ spec:
     spec:
       containers:
       - name: mapper
-        image: "{{ .Values.base.image }}:{{ .Values.base.imageTag }}"
-        imagePullPolicy: {{ .Values.base.imagePullPolicy | quote }}
+        image: "{{ .Values.mapper.repository }}:{{ .Values.mapper.imageTag }}"
+        imagePullPolicy: {{ .Values.mapper.imagePullPolicy | quote }}
         command: ["gosu", "lega", "ega-id-mapper"]
         env:
+        - name: DEFAULT_LOG
+          value: "debug"
         - name: BROKER_CONNECTION_ATTEMPTS
           value: {{ .Values.config.broker_connection_attempts | quote }}
+        - name: BROKER_ENABLE_SSL
+          value: {{ .Values.config.broker_enable_ssl | quote }}
+        - name: BROKER_HEARTBEAT
+          value: {{ .Values.config.broker_heartbeat | quote }}
         - name: BROKER_HOST
           value: {{ if .Values.config.broker_host }}{{ .Values.config.broker_host | quote }}{{ else }}{{ template "localega.fullname" . }}-mq{{ end }}
+        - name: BROKER_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "localega.name" . }}-mq-pass
+              key: password
         - name: BROKER_PORT
           value: {{.Values.mq.port | quote }}
         - name: BROKER_RETRY_DELAY
           value: {{ .Values.config.broker_retry_delay | quote }}
         - name: BROKER_USERNAME
           value: {{ .Values.config.broker_username | quote }}
+        - name: BROKER_VHOST
+          value: "/"
         - name: POSTGRES_HOST
           value: {{ if .Values.config.postgres_host }}{{ .Values.config.postgres_host | quote }}{{ else }}{{ template "localega.fullname" . }}-db{{ end }}
         - name: POSTGRES_PASSWORD
           valueFrom:
             secretKeyRef:
               name: {{ template "localega.name" . }}-db-secret
               key: postgres_password
+        - name: POSTGRES_SSLMODE
+          value: "disable"
```

Comment:
maybe a  miss on this: `{{ .Values.config.broker_enable_ssl | quote }}`

Reply:
My bad ... I think I meant to write this: `{{ .Values.config.postgres_sslmode | quote }}` in `ingest-deploy` I think it should be everywhere the same.